nvflare/app_common/storages/filesystem_storage.py

```diff
@@ -27,20 +27,20 @@
 
 
 def _write(path: str, content):
-    tmp_name = "_" + str(uuid.uuid4())
+    tmp_path = "_" + str(uuid.uuid4())
```

Comment:
Should it be `tmp_path = path + "_" + str(uuid.uuid4())` ?

Reply:
good catch. fixed,